### PR TITLE
virtualpg 2.0.1

### PR DESCRIPTION
**Pull request by fxcoudert**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
The `COPYING` file contains the Mozilla Public License Version 1.1 so I think the license should be `MPL-1.1`.

File: Formula/virtualpg.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Virtualpg < Formula
   desc "Loadable dynamic extension for SQLite and SpatiaLite"
   homepage "https://www.gaia-gis.it/fossil/virtualpg/index"
-  url "https://www.gaia-gis.it/gaia-sins/virtualpg-1.0.2.tar.gz"
-  sha256 "37abd56b2899c90b355e50d843ba019af540b83e7af1da110a766f041abb514e"
+  url "https://www.gaia-gis.it/gaia-sins/virtualpg-2.0.1.tar.gz"
+  sha256 "be2aebeb8c9ff274382085f51d422e823858bca4f6bc2fa909816464c6a1e08b"
```
Comment (Moisan) [inline Formula/virtualpg.rb:6]:
```suggestion
  license `MPL-1.1`
```

**Review by chenrui333 — APPROVED** [review]

File: Formula/virtualpg.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/virtualpg.rb:6]: let's do this in a separate PR along with livecheck as well :)

Comment (chenrui333) [reply Formula/virtualpg.rb:6]: #69101

Comment (chenrui333) [reply Formula/virtualpg.rb:6]: Thanks @Moisan!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**